ci-operator/step-registry/ipi/conf/azure/ipi-conf-azure-ref.yaml

```diff
@@ -1,9 +1,9 @@
 ref:
   as: ipi-conf-azure
   from_image:
-   namespace: origin
-   name: centos
-   tag: '8'
+   namespace: ocp
+   name: "4.5"
+   tag: upi-installer
```

Comment:
Not sure why we want `upi-installer` - and from 4.5 specifically

Reply:
this image contains oc , which is used to get version from the image version "oc adm release info ${release_image}

Reply:
@MayXuQQ I guess @vrutkovs was suggesting like this:
```
ref:
  as: ipi-conf-azure
  from: upi-installer
  cli: latest
```

Reply:
Lets use `from: cli` here, if possible. 

I don't think we ship `upi-installer` in the payload, so when its truly needed we have to import upi-installer from 4.5 imagesteram, but its not the case here imo

Reply:
@vrutkovs thanks for the info. 

When tried `upi-installer` executor images, the following job have different behaviors. 

https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/29967/rehearse-29967-periodic-ci-openshift-hypershift-main-periodics-e2e-conformance-azure/1554722988799037440  failed with reason ImagePullBackOff and message Back-off pulling image "image-registry.openshift-image-registry.svc:5000/ci-op-p4gxyn88/stable:upi-installer"

https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/29967/rehearse-29967-periodic-ci-openshift-verification-tests-master-installer-rehearse-4.12-installer-rehearse-azure/1554662529622347776 get passed, did not hit such problem, do you know what caused the different results.

Reply:
First one is invoked from nightly release - `Resolved release latest to registry.ci.openshift.org/ocp/release:4.11.0-0.ci-2022-08-02-023116` - which doesn't have `upi-installer` included as its not part of the payload.

Second one is invoked for installer, so every installer image - installer, baremetal-installer, upi-installer - are being built and tagged into `stable`. We should avoid using `upi-installer` in periodics as it may not exist in the test payload

Reply:
Got it, thanks a lot.

Reply:
@vrutkovs per the latest test result, seem like cli image does not install `jq`, so maybe we still go back to upi-install:4.5

Reply:
You can import `cli-jq`:
```
  cli-jq:
    name: cli-jq
    namespace: ocp
    tag: latest
```